meta/autoid/autoid.go

```diff
@@ -80,7 +80,11 @@ type Allocator interface {
 	// Alloc allocs N consecutive autoID for table with tableID, returning (min, max] of the allocated autoID batch.
 	// It gets a batch of autoIDs at a time. So it does not need to access storage for each call.
 	// The consecutive feature is used to insert multiple rows in a statement.
-	Alloc(tableID int64, n uint64) (int64, int64, error)
+
+	// increment & offset is used to valid the start position (the allocator's base is not always the last allocated id).
```

Comment:
The function comment begins with the function name. This funcion comment like `Alloc ...`.

Reply:
addressed.